packages/components/psammead-section-label/src/index.jsx

```diff
@@ -90,8 +105,15 @@ Title.propTypes = {
   script: shape(scriptPropType).isRequired,
 };
 
-const SectionLabel = ({ bar, children: title, dir, labelId, script }) => (
-  <Wrapper script={script} bar={bar}>
+const SectionLabel = ({
+  bar,
+  children: title,
+  dir,
+  labelId,
+  script,
+  hideSectionHeader,
```

Comment:
`hideSectionHeader` is a bit odd IMO, as it implies it only hides part of this component. Could it maybe be `visuallyHidden`?